DATA_FORMAT.md

```diff
@@ -51,6 +51,7 @@ Each GeoJSON feature will have a `properties` object with as many of the followi
 | `located_in`          | The name of the feature that this feature is located in.
 | `located_in:wikidata` | The [Wikidata](https://www.wikidata.org/wiki/Wikidata:Main_Page) [item ID](https://www.wikidata.org/wiki/Help:Items) for the brand or chain of the feature that this feature is located in. This is a machine-readable identifier counterpart for the human-readable `located_in` above.
 | `nsi_id`              | The [Name Suggestion Index](https://nsi.guide/) (NSI) ID for the feature. NSI IDs aren't stable, so you may require [old NSI data](https://github.com/osmlab/name-suggestion-index/tree/main/dist) if you are working with old ATP data.
+| `end_date`              | `end_date=yes` is applied when given location is closed and not operating
```

Comment:
It's actually year-month-day or `yes` if unknown. Spec was decided in https://github.com/alltheplaces/alltheplaces/pull/6042

I think we also talked about things closing in the future, although I don't think there is any data yet.

https://github.com/alltheplaces/alltheplaces/blob/156ec96c13d8dd040b617bf73791b625bec126dd/locations/items.py#L85

Reply:
> `end_date=yes` is applied when given location is closed at unknown date and can be assumed to not operate right now, `end_date` may also have values in year-month-day format, including future dates for planned closures.

Though if that is not just for one spider, temporarily, I would consider lifecycle prefix or something else not involving "and this extra field inverts meaning".

Reply:
> > `end_date=yes` is applied when given location is closed at unknown date and can be assumed to not operate right now, `end_date` may also have values in year-month-day format, including future dates for planned closures.
> 
> ?

LGTM, thanks.

Reply:
done now